src/main.c

```diff
@@ -709,9 +717,8 @@ static bool probe_session(yubihsm_context *ctx, int index) {
 }
 
 #ifdef __WIN32
-static void WINAPI timer_handler(void *lpParam __attribute__((unused)),
-                                 unsigned char TimerOrWaitFired
-                                 __attribute__((unused))) {
+static void WINAPI timer_handler(void *lpParam,
+                                 unsigned char TimerOrWaitFired) {
```

Comment:
We should still mark these as unused, otherwise we'll get a warning when compiling?

Reply:
Fixed with UNUSED macro